.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "scripts/nb-tester"
+    schedule:
+      interval: "weekly"
```

Comment:
We might want daily to catch new releases of Qiskit sooner? It's important the docs don't have a big gap where they don't work with new releases.

Reply:
```suggestion
      interval: "daily"
```